portfolio/src/main/webapp/hobbies.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html>
```

Comment:
Check out https://google.github.io/styleguide/htmlcssguide.html#Optional_Tags and consider removing some optional tags!

Reply:
Done! Not sure if its entirely correct (did read through the whole section in the style guide, but I had some doubts). Still, visualization was correct.